### PR TITLE
[10.0][FIX] product_variant_supplierinfo: resolve product variant field writing problem

### PR DESCRIPTION
**Pull request by SimoneVagile**
Discard field override in view
Change invisible attribute

### CODE REVIEW

**Comment by tafaRU** [comment]
@SimoneVagile two of the Travis builds failed due to the following error:

image
I try to restart them, let's see if this will work.

**Comment by SimoneVagile** [comment]
Travis build failed for reasons not imputable to this request.
See [https://travis-ci.org/github/OCA/product-variant/builds/621789865](url)

File: product_variant_supplierinfo/views/product_supplierinfo_view.xml [hunk 1]
```diff
@@ -5,10 +5,8 @@
     <field name="inherit_id" ref="product.product_supplierinfo_form_view"/>
     <field name="model">product.supplierinfo</field>
     <field name="arch" type="xml">
-        <field name="product_name" position="before">
-            <field name="product_id"
-                   domain="[('product_tmpl_id', '=', parent.id)]"
-                   invisible="context.get('hide_product_id')"/>
+        <field name="product_id" position="attributes">
+            <attribute name="invisible">context.get('hide_product_id')</attribute>
```
Comment (tafaRU) [inline product_variant_supplierinfo/views/product_supplierinfo_view.xml:9]: why did you remove the domain?

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: product_variant_supplierinfo/views/product_supplierinfo_view.xml (hunk 1 above)
Comment (SimoneVagile) [reply product_variant_supplierinfo/views/product_supplierinfo_view.xml:9]: Domain is inherited from father view.
From product.product_view:
`<field name="product_id" domain="[('product_tmpl_id', '=', product_tmpl_id)]" invisible="1"/>`

Comment (tafaRU) [reply product_variant_supplierinfo/views/product_supplierinfo_view.xml:9]: Ok, thanks!

**Review by tafaRU — APPROVED** [review]
